mintapi/signIn.py

```diff
@@ -582,6 +582,23 @@ def search_mfa_method(driver):
     return mfa_token_input, mfa_token_button, mfa_method
 
 
+def check_mfa_method(driver):
```

Comment:
I'm not quite sure I understand what this method addresses in the flow? Do you have any screenshots you could add to the PR?

Reply:
This is what I get.  I only have one MFA method, so that I can't specify email as the MFA method.

[Intuit Accounts - Sign In.pdf](https://github.com/mintapi/mintapi/files/9104532/Intuit.Accounts.-.Sign.In.pdf)

Reply:
@tsdg112 this is just the mfa_selection page, so I'd expect you to just pass mfa_method of email so that you get to the mfa_selection_page. If that's not working correctly, then we should fix that method instead. Otherwise, there is blind acceptance of the first one listed, but that not actually be the case.

Reply:
The trouble is that Mint does not always present this screen.  I tried specifying the mfa_method, but when the screen is not presented, mintapi throws MFAMethodNotAvailableError.

Reply:
@tsdg112 is there a way to first check if this screen exists by looking for the header text? Then, if it's there, we validate that mfa_method was supplied. If it's not there, then we proceed as usual.